action.yml

```diff
@@ -0,0 +1,40 @@
+name: "octobors"
+description: "Automatically merge"
+inputs:
+  needs-description-label:
+    description: "Label applied to PRs that don't have a description"
+    required: false
+  required-statuses:
+    description: "List of `,` separated statuses the are required to pass for automerge"
```

Comment:
Can this be inferred from the branch protection rules? Would be nice to not duplicate this information. Not a big deal though.

Reply:
This was meant to work in a scenario where you may not want to enable the status check protections in Github.